api_tests/providers/registrations/views/test_registration_provider_schemas.py

```diff
@@ -71,6 +71,9 @@ def provider_with_v2_prereg_only(self, schema):
     def provider_with_egap_only(self, egap_schema):
         provider = RegistrationProviderFactory()
         update_provider_auth_groups()
+        egap_schema.visible = True
```

Comment:
Drive-by: Hmm. I thought we ended up changing this view so it did not require the schema to be visible. Maybe I thought about it, but it didn't seem worth the change. For the `/v2/schemas/registrations/` endpoint, I would expect that we would need the schema to be visible, but for here, it shouldn't need that.

Reply:
That endpoint only delivers visible schemas, IMO it's better just to make the egap schema visible then to make the special casing special-er. It won't turn up any place import because it's only associated with the EGAP provider.

Reply:
The `visible` attribute only makes sense for `/v2/schemas/registrations`. We have another method of choosing what schemas are visible in this view (i.e. explicitly choosing them for the provider). When we end up adding other providers, we're going to want them not to show in `/v2/schemas/registrations`, because they won't be for the OSF provider using the existing workflow. They'll only be for the new providers. 

We should make this endpoint show regardless of visibility, but keep the other one working as it is.